### ISSUE TITLE
About release 0.28.1

### ISSUE THREAD

**Issue by chenrui333** [kubecfg/kubecfg#156]
👋 It looks like 0.28.1 did not get marked as the latest stable release, raising this issue to confirm if that is intentional. Thanks!

**Comment by mkmik** [comment]
Yeah I wanted to first manually check https://github.com/kubecfg/kubecfg/issues/151#issuecomment-1337107400
